lib/linter/linter.js

```diff
@@ -1711,8 +1711,8 @@ class Linter {
      * @param {string|SourceCode} textOrSourceCode The source code.
      * @param {FlatConfigArray} configArray The config array.
      * @param {VerifyOptions&ProcessorOptions} options The options.
-     * @param {boolean} [firstCall=false] Indicates if this is being called directly
```

Comment:
It's doing this in a bunch of place. I'd file an issue asking for clarification around this behavior because in this case the code DOES specify a default. I can see removing the default from JSDoc when there isn't one in the code, but I don't see why it's being removed here.